### DISCUSSION TITLE
Help about Moose example for thermal creep behavior

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#18318] [Q&A Modules: Solid mechanics] [answered]
Hi Moose experts or users,

I know Moose is a powerful tool to study the thermal creep behavior of alloys. Could anyone share the Moose code for this paper (https://link.springer.com/article/10.1007/s11661-017-4011-3) or the relevant Moose examples with me? It would be helpful for me to learn more about how to achieve the constitutive model presented in the paper in Moose. 
I am appreciative of your help.
@dewenyushu

**Comment by hugary1995** [comment, marked as answer]
You should contact the corresponding author of that paper insteading of asking here.
